config/core/resources/cloudbuildsource.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
   annotations:
     registry.knative.dev/eventTypes: |
       [
-        { "type": "com.google.cloud.build.event", "description": "This event is sent when your build's state changes, such as when your build is created, when your build transitions to a working state, and when your build completes."}
+        { "type": "google.cloud.cloudbuild.build.v1.statusChanged", "description": "This event is sent when your build's state changes, such as when your build is created, when your build transitions to a working state, and when your build completes."}
```

Comment:
Can you follow the example of CloudPubSubSource https://github.com/google/knative-gcp/blob/774f94eb84b8706255c824f84a786295cdf64b92/config/core/resources/cloudpubsubsource.yaml#L25 here to add the schema? The path of schema may change soon because of this PR  googleapis/google-cloudevents#57. You can wait for one day to see whether it gets merged tomorrow.